tiproxy/tiproxy-command-line-flags.md

```diff
@@ -27,28 +27,36 @@ summary: 了解 TiProxy 的命令行参数。
 
 ## TiProxy Control
 
-本节介绍 TiProxy 客户端程序 `tiproxyctl` 的语法、选项和命令。
+本节介绍 TiProxy 客户端程序 `tiproxyctl` 的安装方式、语法、选项和命令。
 
-### 获取 TiProxy Control
+### 安装 TiProxy Control
 
-#### 通过 TiUP 安装
+本节提供两种方式安装 TiProxy Control。
 
-在安装了 TiUP 之后，可以使用 `tiup install tiproxy` 命令来获取 TiProxy 和 TiProxy Control 的二进制程序。TiProxy 的路径可通过 `tiup --binary tiproxy` 查看，TiProxy Control 与 TiProxy 在同一个目录下。
+#### 使用 TiUP 安装
+
+在安装 [TiUP](/tiup/tiup-overview.md) 之后，可以使用 `tiup install tiproxy` 命令获取 TiProxy 和 TiProxy Control 的二进制程序。你可以通过 `tiup --binary tiproxy` 查看 TiProxy 的安装路径，TiProxy Control 与 TiProxy 位于同一目录。
 
 例如：
 
 ```shell
-> tiup install tiproxy
-download https://tiup-mirrors.pingcap.com/tiproxy-v1.3.0-linux-amd64.tar.gz 22.51 MiB / 22.51 MiB 100.00% 13.99 MiB/s
-> ls `tiup --binary tiproxy`ctl
-/root/.tiup/components/tiproxy/v1.3.0/tiproxyctl
+tiup install tiproxy
+# download https://tiup-mirrors.pingcap.com/tiproxy-v1.3.0-linux-amd64.tar.gz 22.51 MiB / 22.51 MiB 100.00% 13.99 MiB/s
+ls `tiup --binary tiproxy`ctl
+# /root/.tiup/components/tiproxy/v1.3.0/tiproxyctl
 ```
 
 #### 从源代码编译安装
 
 编译环境要求：[Go](https://golang.org/) 1.21 或以上版本。
 
-编译步骤：在 [TiProxy 项目](https://github.com/pingcap/tiproxy)根目录，使用 `make` 命令进行编译，生成 tiproxyctl。
+编译步骤：在 [TiProxy 项目](https://github.com/pingcap/tiproxy)根目录，使用 `make` 命令进行编译，生成 `tiproxyctl`。
+
+```shell
+git clone https://github.com/pingcap/tiproxy.git
+cd tiproxy
+make
```

Comment:
Please add a command that allows users to verify if `tiproxyctl` has been installed successfully.